lang/en.yml

```diff
@@ -93,7 +125,7 @@ en:
     VIEWINGVERSION: 'Currently viewing version {version}.'
     WHEN: When
   SilverStripe\CMS\Controllers\CMSPageSettingsController:
-    MENUTITLE: 'Edit Page'
+    MENUTITLE: Pages
```

Comment:
Is this supposed to change?

Reply:
See https://github.com/silverstripe/silverstripe-cms/pull/3080#discussion_r2092124290